documentation/sdk/docs/_admonitions/_destroy_foundry.md

```diff
@@ -0,0 +1,5 @@
+:::warning Destroying Foundry
+
+A foundry can only be destroyed (and it's storage deposit be claimed) if the circulating supply is zero. Therefore if you control the foundry and want to destroy it in the future, you should consider [melting your native tokens](../how_tos/native_tokens/decrease_supply.mdx) instead of [burning](../how_tos/native_tokens/burn.mdx) them.
```

Comment:
```suggestion
A foundry can only be destroyed (and its storage deposit be claimed) if the circulating supply is zero. Therefore if you control the foundry and want to destroy it in the future, you should consider [melting your native tokens](../how_tos/native_tokens/decrease_supply.mdx) instead of [burning](../how_tos/native_tokens/burn.mdx) them.
```